scripts/setup.sh

```diff
@@ -6,6 +6,8 @@ cd ../
 
 bun i
 
+make all-js
```

Comment:
I think the goal is to not rely on make anymore. 
There's no cmake equivalent?

Reply:
Hey again jerome,

If you look at the pull request list you'll see I already have opened a PR moving runtime js and fallback decoder to CMake. Dave is working on adding bun_error and node-fallbacks to the scripts. The use of make command here is meant to be a hotfix till those changes are integrated, otherwise contributors won't be able to build the project till those changes.

Thank you